Set/Game/SetGameViewModel.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import SwiftUI
+
+class SetGameViewModel: ObservableObject {
+    @Published private var gameModel: SetGameModel
+
+    init() {
+        gameModel = SetGameModel()
```

Comment:
Here is an issue that SetGameModel doesn't accept anything in init but later it's changed in `createDeckOfCards` function. 
I would recommend make a `Builder` that return array of `SetGameModel.Card` and make SetGameModel accept a "deck" (e.g. array of card in init).
Anyway, this approach is ok, but it can be improved with some best practices.
...
in case when you have to change `gameModels`s cards array when you hit "new game" button it can have `func set(deck: [SetGameModel.Card]) which will replace existing value in `cards` property and don't just append it.

Reply:
We can have steps like these:
1. create deck (array of cards) with special builder class.
2. create new game model with predefined set of cards.